.gitignore

```diff
@@ -28,4 +28,4 @@ report
 # On development branch only. This should be removed for point releases
 Cargo.lock
 
-common/log/
+*.log
```

Comment:
This will ignore empty log directories - log directories can currently appear in any directory that the code is run.